.swiftlint.yml

```diff
@@ -117,7 +117,7 @@ identifier_name:
     - z
 
 indentation_width:
-  indentation_width: 2
+  indentation_width: 4
```

Comment:
I wanted to bring this one to your attention @pacu.  I saw we had 2 previously in the SwiftLint file but 4 is Xcodes default.  Let me know if this makes sense or if you feel that we should re work this to go with 2.

Reply:
that's fine. good catch